bondcpp/include/bondcpp/bond.hpp

```diff
@@ -134,19 +161,23 @@ class Bond
    * \return true iff the bond has been formed.
    */
   bool waitUntilFormed(rclcpp::Duration timeout = rclcpp::Duration(std::chrono::seconds(-1)));
+
   /** \brief Blocks until the bond is broken for at most 'duration'.
    *    Assumes the node to be spinning in the background
    *
    * \param timeout Maximum duration to wait.  If -1 then this call will not timeout.
    * \return true iff the bond has been broken, even if it has never been formed.
    */
   bool waitUntilBroken(rclcpp::Duration timeout = rclcpp::Duration(std::chrono::seconds(-1)));
+
   /** \brief Indicates if the bond is broken.
    */
   bool isBroken();
+
   /** \brief Breaks the bond, notifying the other process.
    */
   void breakBond();
+
   std::string getTopic() {return topic_;}
   std::string getId() {return id_;}
   std::string getInstanceId() {return instance_id_;}
```

Comment:
Since we are making changes here anyway, these can all be `const` methods.

Reply:
Done